mozci/task.py

```diff
@@ -62,6 +62,18 @@ class Status(Enum):
 )
 
 
+def get_configuration_from_label(label: str) -> str:
```

Comment:
Maybe you can refer to bug 1632870 here so if we end up fixing that and stumble across this comment, we can create an issue to switch over.

Reply:
Done!